packages/react-components/src/components/styles/EditBox.styles.ts

```diff
@@ -40,6 +33,6 @@ export const inputBoxIcon = mergeStyles({
  */
 export const editBoxStyleSet = {
   root: {
-    width: '100%'
+    minWidth: '30vw'
```

Comment:
dont use vw - the composite may or may not be 100% of the viewport width/height so this likely doesn't behave as expected when the composite is < 100% of the width of the webpage

Reply:
What would you suggest here for layout of the edit box @JamesBurnside ?